### PR TITLE
Neural Graph Differential Equation Example

### PR DESCRIPTION
**Pull request by avik-pal**
This modifies the GCN demo in GeometricFlux to use a Neural GDE.

https://github.com/yuehhua/GeometricFlux.jl/pull/62 needs to be merged and a release has to be tagged before it works properly.

### CODE REVIEW

**Comment by yuehhua** [comment]
Thank you for implement this example. I just got everything done at yuehhua/GeometricFlux.jl#64.
I will tag it as new release ASAP.
Related issue #80

**Comment by yuehhua** [comment]
Wait for new release JuliaRegistries/General#17623

**Comment by ChrisRackauckas** [comment]
Let's get a test added that will capture errors in the tutorial. That'll make it easier to maintain over time. Probably just a simple example where a NGDE is trained on some silly loss and just make sure the loss goes down.

**Comment by yuehhua** [comment]
I just check travis-ci, and it shows the following words:
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
```
It seems that it failed because the build time is too long and timeout happens. Might not due to the code itself but I am not sure about the root cause. Whenever I found that the CI failed not because of my code, I just restart it and it get passed.

**Comment by ChrisRackauckas** [comment]
Yeah, Julia v1.4 is the high of `using` times... at least v1.5 fixes it.

File: test/neural_gde.jl [hunk 1]
```diff
@@ -0,0 +1,35 @@
+using OrdinaryDiffEq, Flux, Test, GeometricFlux
```
Comment (ChrisRackauckas) [inline test/neural_gde.jl:1]:
```suggestion
using OrdinaryDiffEq, DiffEqFlux, Flux, Test, GeometricFlux
```

**Comment by ChrisRackauckas** [comment]
Completed the test change in #355 and merged these commits. Thanks!

**Pull request closed without merging**